libgdf/src/tests/groupby/groupby-test.cu

```diff
@@ -116,15 +118,19 @@ struct GroupTest : public GdfTest {
 
     // Create a new instance of a gdf_column with a custom deleter that will free
     // the associated device memory when it eventually goes out of scope
-    auto deleter = [](gdf_column* col){col->size = 0; rmmFree(col->data, 0);};
+    auto deleter = [](gdf_column* col){col->size = 0; rmmFree(col->data, 0); rmmFree(col->valid, 0); };
     gdf_col_pointer the_column{new gdf_column, deleter};
 
     // Allocate device storage for gdf_column and copy contents from host_vector
     rmmAlloc(&(the_column->data), host_vector.size() * sizeof(col_type), 0);
     cudaMemcpy(the_column->data, host_vector.data(), host_vector.size() * sizeof(col_type), cudaMemcpyHostToDevice);
 
+    int valid_size = gdf_get_num_chars_bitmask(host_vector.size());
+    rmmAlloc((void**)&(the_column->valid), valid_size, 0);
```

Comment:
After latest RMM merge, you need to use RMM_ALLOC. Also, there is no error checking in this function.  Need to use RMM_TRY, but that requires this function to return an error.

Reply:
Perhaps error checking should be a follow up PR. I will file an issue. For now, just replace rmmAlloc with RMM_ALLOC.

Reply:
I have added EXPECT_EQ for error checking.